src/WebSdk/Publish/Targets/Microsoft.NET.Sdk.Publish.targets

```
@@ -36,6 +36,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
              And '$(RuntimeIdentifier)' != ''
              And !$(RuntimeIdentifier.StartsWith('win'))
              And '$(PublishIISAssets)' == ''">false</PublishIISAssets>
+    <PublishIISAssets Condition="'$(PublishAot)' == 'true' And '$(PublishIISAssets)' == ''">false</PublishIISAssets>
```

```suggestion
    <PublishIISAssets Condition="'$(PublishIISAssets)' == '' and '$(PublishAot)' == 'true'">false</PublishIISAssets>
```

It reads better to put the "has this already been set?" check first.